core/common/src/main/java/alluxio/util/ConfigurationUtils.java

```diff
@@ -102,4 +103,13 @@ public static Properties searchPropertiesFile(String propertiesFile,
     return loadPropertiesFromResource(propertiesFile);
   }
 
+  /**
+   * @return whether the configuration describes how to find the master host, either through
+   *         explicit configuration or through zookeeper
+   */
+  public static boolean masterHostConfigured() {
+    boolean usingZk = Configuration.getBoolean(PropertyKey.ZOOKEEPER_ENABLED)
```

Comment:
This may not be related but do you know if there is a check somewhere that verifies that both `ZOOKEEPER_ENABLED` and `ZOOKEEPER_ADDRESS` are set together?